### DISCUSSION TITLE
Scaling on Linux on Ultrawide or 4k Monitors

### DISCUSSION THREAD

**Discussion by boospy** [rustdesk/rustdesk#4697] [General]
Version 1.2 is very tiny on big resolutions. For this you can set an ENV before: 

`env GDK_SCALE=2 rustdesk`

Yes i is a little bit big, but better then tiny.

**Comment by hack-wizard** [comment]
Additional context here - if you're editing a launcher, make sure you add GDK_SCALE=2 to environment vairables, NOT in front of the command.  It will not work otherwise, found that out the hard way when editing my launcher
